ci-operator/step-registry/hypershift/mce/power/heterogeneous/x86-power/hypershift-mce-power-heterogeneous-x86-power-workflow.yaml

```diff
@@ -0,0 +1,30 @@
+workflow:
+  as: hypershift-mce-power-heterogeneous-x86-power
```

Comment:
I don't think we need to add a new workflow. This workflow is very similar to hypershift-mce-power-conformance, with only the difference being:
```
chain: hypershift-mce-power-create-ibmcloud-workers. 
```
So, we can include it in the test.
E.g
```
- as: e2e-x86-power-heterogeneous-conformance
  cron: 0 8 * * *
  steps:
    cluster_profile: aws
    env:
      MCE_VERSION: "2.7"
      ADDITIONAL_HYPERSHIFT_NODEPOOL_CONFIG: |
        NAME_SUFFIX=power, CPU_ARCH=ppc64le, NODE_ARCH=ppc64le
        NAME_SUFFIX=x86, CPU_ARCH=x86_64, NODE_ARCH=amd64
    test:
    - chain: hypershift-mce-power-create-ibmcloud-workers
    - chain: hypershift-mce-power-test
    - ref: hypershift-mce-agent-info
    workflow: hypershift-mce-power-conformance
```

Reply:
Thanks for the review.
Merged it into hypershift-mce-power-conformance workflow